task19/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <script type="module" src="js/slider.js" defer></script>
+    <script type="module" src="js/news.js" defer></script>
+    <title>JS課題19</title>
+  </head>
+  <body class="p-16 bg-gray-300 font-sans tracking-widest">
+    <div id="js-slideShowContainer" class="slideshow-container max-w-full mx-auto mb-12">
+      <div id="js-slideShowItem" class="slideshow-item flex justify-between items-center">
+        <div id="js-slideShowWrapper" class="slideshow-wrapper w-8/12 h-96 mx-auto bg-white list-none relative overflow-hidden">
```

Comment:
nit
`list-style: none;` is unnecessary because it is loaded in base.
```suggestion
        <div id="js-slideShowWrapper" class="slideshow-wrapper w-8/12 h-96 mx-auto bg-white relative overflow-hidden">
```

Reply:
Thanks for looking at all the details. Fixed it!
[91b6fb3](https://github.com/seeeen044/JavaScript_handson/pull/20/commits/91b6fb37450f1acca6d7b58b73f02b0b037fbe0b)